src/spell_check.c

```diff
@@ -173,21 +173,30 @@ RS_Suggestion **spellCheck_GetSuggestions(RS_Suggestions *s) {
   return ret;
 }
 
+static double spellCheck_GetMaxScore(arrayof(RS_Suggestion) *suggestions) {
```

Comment:
`arrayof(X)` is "array of objects of type X", which is actually a X*.
Here X=RS_Suggestion* (this is an array of pointers), so it should be `arrayof(RS_Suggestion*) suggestions` (which maps to `RS_Suggestions **`, just like the original).